src/datajunction/sql/build.py

```diff
@@ -89,13 +91,20 @@ def get_filter(columns: Dict[str, SqlaColumn], filter_: str) -> BinaryExpression
     if name not in columns:
         raise Exception(f"Invalid column name: {name}")
     column = columns[name]
-
-    try:
-        value = ast.literal_eval(value)
-    except Exception as ex:
-        raise Exception(f"Invalid value: {value}") from ex
-
     comparison = COMPARISONS[operator_]
+
+    if type(column.type) in [Date, DateTime]:
```

Comment:
Eventually I think we'll need to use the Python type here:

```python
if ColumnType(column.type.python_type.upper()) in {ColumnType.DATE, ColumnType.DATETIME}:
```

Since I think some databases defined their own specific types for Date and DateTime.

Reply:
Done.